docs/source/developer_guide/index.md

```diff
@@ -1,5 +1,11 @@
 # Developer Guide
 
+cuSpatial has two main components: the cuSpatial Python package and the `libcuspatial` C++ library,
+referred to as `cuspatial` and `libcuspatial` respectively in this documentation. This page
+discusses the design of `cuspatial`. For information on `libcuspatial`, see the [libcuspatial
+developer guide](https://docs.rapids.ai/api/libcuspatial/nightly/DEVELOPER_GUIDE.html)
```

Comment:
Please link to the stable... 

Is there a way to make these links relative so that the nightly Python user guide links to the nightly C++ user guide, and the stable Python user guide links to the stable C++ user guide?
```suggestion
developer guide](https://docs.rapids.ai/api/libcuspatial/stable/DEVELOPER_GUIDE.html)
```

Reply:
At least not from the sphinx docs. You would probably need to add some scripts to the nightly build system?

Reply:
OK then just always link to the stable docs.